### PR TITLE
Add Agent ECS-Anywhere support doc

### PR DESCRIPTION
**Pull request by clamoriniere**
### What does this PR do?

Add Agent ECS-Anywhere support doc, the information are extracted from the https://www.datadoghq.com/blog/amazon-ecs-anywhere-monitoring/ blogpost.

### Preview

https://docs-staging.datadoghq.com/clamoriniere/aws-ecs-anywhere/agent/amazon_ecs/?tab=awscli

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/amazon_ecs/_index.md [hunk 1]
```diff
@@ -66,7 +66,13 @@ Configure the task using either the [AWS CLI tools][12] or using the Amazon Web
 
 1. For Linux containers, download [datadog-agent-ecs.json][1] ([datadog-agent-ecs1.json][2] if you are using an original Amazon Linux 1 AMI). For Windows, download [datadog-agent-ecs-win.json][3].
 2. Edit `datadog-agent-ecs.json` and set `<YOUR_DATADOG_API_KEY>` with the [Datadog API key][4] for your account.
-3. Optionally - Add an Agent health check.
+3. Optionally - Add the following to you ECS task definition to deploy on an ["ECS-Anywhere" cluster][5].
```
Comment (ahmed-mez) [inline content/en/agent/amazon_ecs/_index.md:69]: - just to double check, Is this optional or required on `"ECS-Anywhere" cluster` ? 
- could we also update the `Web UI` tab accordingly?

Comment (clamoriniere) [reply content/en/agent/amazon_ecs/_index.md:69]: Require for ECS-A

Comment (clamoriniere) [reply content/en/agent/amazon_ecs/_index.md:69]: Added the information on the `Web UI` tab.

**Review by ahmed-mez — APPROVED** [review]

File: content/en/agent/amazon_ecs/_index.md (hunk 1 above)
Comment (kayayarai) [reply content/en/agent/amazon_ecs/_index.md:69]:
```suggestion
3. Optionally - Add the following to your ECS task definition to deploy on an [ECS Anywhere cluster][5].
```

File: content/en/agent/amazon_ecs/_index.md [hunk 2]
```diff
@@ -95,14 +101,16 @@ aws ecs register-task-definition --cli-input-json <path to datadog-agent-ecs.jso
...
+[5]: https://www.datadoghq.com/blog/amazon-ecs-anywhere-monitoring/
+[6]: /agent/amazon_ecs/logs/
+[7]: /agent/amazon_ecs/apm/
 {{% /tab %}}
 {{% tab "Web UI" %}}
 
 1. Log in to your AWS Console and navigate to the EC2 Container Service section.
 2. Click on the cluster you wish to add Datadog to.
 3. Click on **Task Definitions** on the left side and click the button **Create new Task Definition**.
+   Select "External" on the launch type step if you plan to deploy the agent task on an "ECS-Anywhere" cluster.
```
Comment (kayayarai) [inline content/en/agent/amazon_ecs/_index.md:113]:
```suggestion
   Select "External" on the launch type step if you plan to deploy the agent task on an ECS Anywhere cluster.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**